data/covering_grammar/lib/error_analysis.py

```diff
@@ -19,51 +19,60 @@
 
 import argparse
 
-import prettytable  # type: ignore
```

Comment:
one late-breaking change: because you removed this, this now fails a `mypy` test. (This directory isn't automatically checked by `mypy`, I don't think, hence it's not flagged on presubmit.) The comment tells it that `prettytable` doesn't yet have type hints and to ignore it in type checking. Could you restore it before we submit?